compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitView.uikit.kt

```diff
@@ -17,166 +17,122 @@
     { rect -> this.view.setFrame(rect) }
 
-/**
- * @param factory The block creating the [UIView] to be composed.
- * @param modifier The modifier to be applied to the layout. Size should be specified in modifier.
- * Modifier may contains crop() modifier with different shapes.
- * @param update A callback to be invoked after the layout is inflated.
- * @param background A color of [UIView] background wrapping the view created by [factory].
- * @param onRelease A callback invoked as a signal that this view instance has exited the
- * composition hierarchy entirely and will not be reused again. Any additional resources used by the
- * View should be freed at this time.
- * @param onResize May be used to custom resize logic.
- * @param interactive If true, then user touches will be passed to this UIView
- * @param accessibilityEnabled If `true`, then the view will be visible to accessibility services.
- *
- * If this Composable is within a modifier chain that merges
- * the semantics of its children (such as `Modifier.clickable`), the merged subtree data will be ignored in favor of
- * the native UIAccessibility resolution for the view constructed by [factory]. For example, `Button` containing [UIKitView]
- * will be invisible for accessibility services, only the [UIView] created by [factory] will be accessible.
- * To avoid this behavior, set [accessibilityEnabled] to `false` and use custom [Modifier.semantics] for `Button` to
- * make the information associated with this view accessible.
- *
- * If there are multiple [UIKitView] or [UIKitViewController] with [accessibilityEnabled] set to `true` in the merged tree, only the first one will be accessible.
- * Consider using a single [UIKitView] or [UIKitViewController] with multiple views inside it if you need multiple accessible views.
- *
- * In general, [accessibilityEnabled] set to `true` is not recommended to use in such cases.
- * Consider using [Modifier.semantics] on Composable that merges its semantics instead.
- *
- * @see Modifier.semantics
- */
+@Deprecated(
+    message = "Use androidx.compose.ui.viewinterop.UIKitView instead"
+)
 @Composable
 fun <T : UIView> UIKitView(
     factory: () -> T,
     modifier: Modifier,
-    update: (T) -> Unit = STUB_CALLBACK_WITH_RECEIVER,
+    update: (T) -> Unit = NoOp,
     background: Color = Color.Unspecified,
-    onRelease: (T) -> Unit = STUB_CALLBACK_WITH_RECEIVER,
+    onRelease: (T) -> Unit = NoOp,
     onResize: (view: T, rect: CValue<CGRect>) -> Unit = DefaultViewResize,
     interactive: Boolean = true,
     accessibilityEnabled: Boolean = true
 ) {
-    val compositeKeyHash = currentCompositeKeyHash
-    val interopContainer = LocalInteropContainer.current
+    // Despite the name, onResize actually contains the logic for default resizing strategy.
+    // Since this strategy is already implied, changes to this argument can't be processed in a
+    // sane manner
+    require(onResize == DefaultViewResize) {
```

Comment:
It seems like a breaking change if it was used previously

Reply:
Yes, but it didn't make sense because it basically required user to set a proper frame of interop view, which was already laid out by compose.

Reply:
It doesn't matter. It can be used in libraries that the user doesn't control, so it will become a blocker for Compose update

Reply:
When did we stabilise this API in the first place?